src/utils/order_util.py

```diff
@@ -0,0 +1,16 @@
+# reducer function to get assigned chef map
```

Comment:
Python docs format is the following way:
```
def functionA():
    ```
   My comments are here
    ```
```

Take a look in the internet for more information about Python conventions for documentation.

Reply:
done.